alma/alma.php

```diff
@@ -122,9 +123,31 @@ public function install()
 
         Tools::clearCache();
 
+        // Enable Alma payment for all installed carriers in PrestaShop 1.7+
+        if (version_compare(_PS_VERSION_, '1.7', '>=')) {
+            $this->updateCarriersWithAlma();
+        }
+
         return $this->installTabs();
     }
 
+    private function updateCarriersWithAlma()
+    {
+
+        $id_module = $this->id;
+        $id_shop = (int) $this->context->shop->id;
+        $id_lang = $this->context->language->id;
+        $carriers = CarrierCore::getCarriers($id_lang, false, false, false, null, CarrierCore::ALL_CARRIERS);
```

Comment:
Use classes without the `Core` suffix

```suggestion
        $carriers = Carrier::getCarriers($id_lang, false, false, false, null, Carrier::ALL_CARRIERS);
```